trino/client.py

```diff
@@ -36,6 +36,9 @@
 import copy
 import os
 import re
+from decimal import Decimal
```

Comment:
nit: ordering of import - non-standard modules should go at the end + alphabetical sorting